tests/run-make/used/Makefile

```diff
@@ -1,7 +0,0 @@
-include ../tools.mk
-
-# ignore-windows-msvc
```

Comment:
Problem: I believe this ignore might be needed, `nm` might not be available on Windows msvc? (Is it then available for -mingw or -gnu?)

Reply:
... It's so obvious when someone is pointing it out. ^^'